src/librustdoc/lib.rs

```diff
@@ -248,8 +247,8 @@ fn opts() -> Vec<RustcOptGroup> {
                 "e",
                 "extend-css",
                 "To add some CSS rules with a given file to generate doc with your \
-                      own theme. However, your theme might break if the rustdoc's generated HTML \
-                      changes, so be careful!",
+                        own theme. However, your theme might break if the rustdoc's generated HTML \
```

Comment:
Wouldn't this cause the text to have double space?

Reply:
As far as I know the `\` above followed by a newline will "eat" all whitespace in the next line(s) until a non-space character, so indenting it more is not an issue.